desktop/core/src/desktop/settings.py

```diff
@@ -38,6 +38,7 @@
 from aws.conf import is_enabled as is_s3_enabled
 from azure.conf import is_abfs_enabled
 from desktop.conf import is_ofs_enabled
+from urllib.parse import urlparse
```

Comment:
If importing like this, then we need to make sure this is not cherrypicked in any Py2 based Hue releases.

Reply:
My understanding is that we have not stopped supporting py2 in the master branches and have also removed some of the py2 code.

Reply:
Yes agreed, mentioned just to be super sure about cherrypicking.